src/main/java/org/apache/datasketches/memory/StepBoolean.java

```diff
@@ -62,7 +62,7 @@ boolean change() {
    * Return true if the state has changed from the initial state
    * @return true if the state has changed from the initial state
    */
-  boolean hasChanged() {
+  public boolean hasChanged() {
     return state != initialState;
   }
```

Comment:
Did you add the public visibility modifier for reflective access from the tests?

Reply:
Yes, and this class may have use cases outside Memory.  There was no reason this method needed to be non-public.